src/main/java/com/converterapp/ConsoleUIProgram.java

```diff
@@ -17,45 +14,53 @@ public class ConsoleUIProgram {
     public static void main(String[] args) throws IOException {
         ConsoleDialog cd = ConsoleDialog.getInstance();
         if (args.length == 0) {
-            Path path = cd.userPromptFilePath();
-            String content = Files.readString(path);
-            if (FileValidator.isXML(content)) {
-                System.out.println(Converter.createXmlDtoFromFileContent(content).getHierarchy());
-            } else if (FileValidator.isJSON(content)) {
-                for (JsonDto json : Converter.createJsonDtoListFromFileContent(content, true)) {
-                    System.out.println(json.getHierarchy());
-                }
-            } else {
-                System.out.println("Content is neither xml nor json");
+            run(cd);
+        } else {
+            test(args, cd);
+        }
+    }
+
+    private static void run(ConsoleDialog cd) throws IOException {
+        Path path = cd.userPromptFilePath();
+        String content = Files.readString(path);
+        if (FileValidator.isXML(content)) {
+            System.out.println(XmlDtoReader.readXmlDto(new StringBuilder(content)).getHierarchy());
+        } else if (FileValidator.isJSON(content)) {
+            for (JsonDto json : JsonDtoReader.readJsonDtoList(new StringBuilder(content), true)) {
+                System.out.println(json.getHierarchy());
             }
         } else {
-            String content;
-            switch (args[0]) {
-                case "stage1test" -> {
-                    content = cd.userPromptLine();
-                    if (FileValidator.isXML(content)) {
-                        System.out.println(Converter.xmlToJSON(content));
-                    } else if (FileValidator.isJSON(content)) {
-                        System.out.println(Converter.jsonToXML(content));
-                    } else {
-                        System.out.println("Content is neither xml nor json");
-                    }
-                }
-                case "stage2test", "stage5test", "stage6test" -> {
-                    content = org.assertj.core.util.Files.contentOf(new File("test.txt"), StandardCharsets.UTF_8);
-                    if (FileValidator.isXML(content)) {
-                        System.out.println(Converter.xmlToJSON(content));
-                    } else if (FileValidator.isJSON(content)) {
-                        System.out.println(Converter.jsonToXML(content));
-                    } else {
-                        System.out.println("Content is neither xml nor json");
-                    }
+            System.out.println("Content is neither xml nor json");
```

Comment:
Do not use hardcoded strings. Create interface AppConctants and put all strings there